int32 camera_id
bool target_find
int32 target_x
int32 target_y
int32 target_z